textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/GramBooleanQuery.java

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
A general comment: in the description of this PR, can you add an example to illustrate what this PR does?  It can help the review a lot.